addons/website/views/assets.xml

```diff
@@ -86,6 +81,12 @@
         <script type="text/javascript" src="/website/static/src/js/show_password.js"/>
         <!-- Custom empty file for user javascript -->
         <script type="text/javascript" src="/website/static/src/js/user_custom_javascript.js"/>
+        <script type="text/javascript" src="/web/static/lib/ace/ace.js"/>
+        <script type="text/javascript" src="/web/static/lib/ace/javascript_highlight_rules.js"/>
+        <script type="text/javascript" src="/web/static/lib/ace/mode-scss.js"/>
+        <script type="text/javascript" src="/web/static/lib/ace/mode-js.js"/>
+        <script type="text/javascript" src="/web/static/lib/ace/mode-xml.js"/>
+        <script type="text/javascript" src="/web/static/lib/ace/theme-monokai.js"/>
     </xpath>
```

Comment:
You can't add those in assets_frontend like that, website visitors don't need those. In fact, how are those loaded for the HTML/CSS editor? Probably lazy loading on the related widget? It should be the same here :+1: